.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: ruby
+cache:
+  directories:
+    - bundle
+
+rvm:
+  - 2.0
+  - 2.1.3
+  - 2.3.1
+
+script:
+    - bundle exec rake spec
+    # - bundle exec danger plugin lint
```

Comment:
I just activated this repo on travis.

![](https://travis-ci.org/dbgrandi/danger-proselint.svg)